src/IngredientEntity.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace GRUB; 
+
+/* Class for IngredientEntities */
+class IngredientEntity {
+    private $name;
+    
+    /**
+     * Constructor for IngredientEntity
+     *
+     * @param string $name Name for ingredient object
+     */
+    public function __construct(string $name){
+
+        $this->name = $name;
+    }
+
+    /**
+     * Returns name of ingredient object as string
+     *
+     * @return string Name of ingredient
+     */
+    public function getName(): string{
+        return $this->name;
+    }
+
+    /**
+     * Generates HTML form checkbox for ingredient
+     *
+     * @return string HTML string
+     */
+    public function generateHTML(): string{
+        $html = "<div class='ingredient'>";
+        $html .= "<input type='checkbox' name=".$this->getName()."/>";
```

Comment:
```suggestion
        $html .= "<input type='checkbox' name=" . $this->getName() . "/>";
```
added space for clarity

Reply:
Its all one block, doesn't make sense to add space